openshift/openshift.go

```diff
@@ -164,7 +164,7 @@ type openshiftImageSource struct {
 	// Values specific to this image
 	sys *types.SystemContext
 	// State
-	docker               types.ImageSource // The Docker Registry endpoint, or nil if not resolved yet
+	docker               types.ImageSource // The container registry endpoint, or nil if not resolved yet
```

Comment:
This is confusing; the OpenShift integrated registry is also a “container registry”, so a “container registry endpoint” is not a distinction.

Refer to the docker/distribution (distribution/distribution? the WIP OCI distribution idea?) protocol here.

Also the other instance.

Reply:
So you want to say the docker/distribution endpoint?

Reply:
“docker/distribution API” I guess

Reply:
Fixed